src/character.h

```diff
@@ -4018,6 +4018,7 @@ class Character : public Creature, public visitable
         int strain;
         int strain_burn;
         bool used_strain_this_turn = false;
+        bool recover_burn_ticker = 4;
```

Comment:
```suggestion
        int recover_burn_ticker = 4;
```

I'm pretty sure this is meant to be an `int` instead.

Reply:
Yes. Given that I've compiled and played this, I'm surprised it didn't come up earlier.